keyboards/3.0/fr-t-k0-azerty.xml

```diff
@@ -52,19 +52,32 @@
 	</displays>
 
 	<keys>
-		<key id="super-2" to="²" longPress="₂" />
-		<key id="amp" to="&amp;" /> <!-- 1 -->
+		<import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml" />
+		<import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+
+		<!-- switch keys -->
+		<key id="shift" switch="shift" />
+		<key id="numeric" switch="numeric" />
+		<key id="symbol" switch="symbol" />
+		<key id="base" switch="base" />
+
+
+		<key id="bksp" gap="true" /> <!-- TODO -->
```

Comment:
Why is bksp being defined as a gap? I don't think we should be able to redefine bksp or enter.